databuilder/example/scripts/sample_json_query_metadata_loader.py

```diff
@@ -0,0 +1,171 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This is a example script which demo how to load data into neo4j without using Airflow DAG.
+"""
+
+import datetime as dt
+import json
+import logging
+import os
+import sys
+import uuid
+
+from elasticsearch.client import Elasticsearch
+from pyhocon import ConfigFactory
+
+from databuilder.extractor.neo4j_extractor import Neo4jExtractor
+from databuilder.extractor.neo4j_search_data_extractor import Neo4jSearchDataExtractor
+from databuilder.extractor.query.json_query_metadata_extractor import JsonQueryMetadataExtractor
```

Comment:
do we still need this sample loader?

Reply:
I think we don't run flake8/mypy on scripts folder, otherwise this file will fail